libAvKys/Plugins/Crop/share/qml/main.qml

```diff
@@ -0,0 +1,328 @@
+/* Webcamoid, webcam capture application. Crop Plug-in.
+ * Copyright (C) 2022  Tj <hacker@iam.tj>
+ *
+ * Webcamoid is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Webcamoid is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Webcamoid. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Web-Site: http://webcamoid.github.io/
+ */
+
+import QtQuick 2.12
+import QtQuick.Controls 2.5
+import QtQuick.Layouts 1.3
+import CropElement 1.0
+
+GridLayout {
+    property real aspectRatio: 0
+    property bool firstMax: true
+    property bool firstMin: true
+    property int tipDelay: 1000
+    property int tipTimeout: 5000
+    property rect source: Qt.rect(0,0,0,0)
```

Comment:
https://github.com/webcamoid/webcamoid/wiki/Coding-style-and-conventions#functions-and-methods